deployment/checkCmd.py

```diff
@@ -2,10 +2,52 @@
 import os
 import logging
 import logging.config
+import yaml
+import re
+from k8sPaiLibrary.maintainlib import common as pai_common
 
 logger = logging.getLogger(__name__)
 
 
+def validate_config(config_path):
+    layout_file = os.path.join(config_path, "layout.yaml")
```

Comment:
Directly read from yaml file? Can you read the machine list or the so called layout from cluster object model? Other wise, if the name of ```machine-list``` is changed, you will have to change 2 places.